apps/ledger-live-mobile/src/components/DeviceAction/index.tsx

```diff
@@ -334,6 +338,31 @@ export function DeviceActionDefaultRendering<R, H extends Status, P>({
     });
   }
 
+  // FIXME when we rework the error rendering, this should be handled at that
+  // level instead of being an exception here.
+  if (imageRemoveRequested) {
+    if (error && (error as Status["error"]) instanceof UserRefusedOnDevice) {
```

Comment:
this if is different from the one on LLD and think the LLD one is the correct one. Cause this will only render the error if the error is of type "UserRefusedOnDevice", otherwise it will render the allow screen. So if, for example, we have a DisconnectedDevice, we'll still be rendering the allow sreen.

Reply:
Not sure I follow